### PR TITLE
Rename AI implementations

### PR DESCRIPTION
**Pull request by neopostmodern**
(Sorry for mingling some concerns – refactor, reformat, ignore file...)

### CODE REVIEW

**Comment by neopostmodern** [comment]
I seem to have linting/formatting issues locally, I'll go through that first...

**Comment by maccinza** [comment]
Shouldn't we merge it into `refactor-to-django`?

**Comment by neopostmodern** [comment]
@maccinza I guess I was assuming that we'd have merged #44 before this, but maybe it's better to do it the other way around 👍

File: mmvb_backend/urls.py [hunk 1]
```diff
@@ -13,9 +13,15 @@
...
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+from django.contrib import admin
+from django.urls import include, path
+from django.views.generic import TemplateView
 from rest_framework.schemas import get_schema_view
 
-from ais.api.urls import router as ais_router
+from ai_implementations.api.urls import router as ai_implementations_router
+from benchmarking_sessions.api.urls import (
```
Comment (maccinza) [inline mmvb_backend/urls.py]: I guess this slipped into but should not be in this PR, right?

Comment (neopostmodern) [reply mmvb_backend/urls.py]: 👀

**Review by maccinza — APPROVED** [review]
LGTM!

**Pull request merged**